src/Reflection/ReflectionClassConstant.php

```diff
@@ -91,6 +91,16 @@ public function getValue()
     }
 
     /**
+     * Get the value represented as a string.
+     *
+     * @return string
+     */
+    public function getValueAsString() : string
```

Comment:
@vlastavesely thanks for your understanding :-)

Also, this is the right time to do it, as this code is not yet in a stable release :+1: